addon/props/model_props.py

```diff
@@ -86,6 +86,21 @@ class SOURCEOPS_ModelProps(bpy.types.PropertyGroup):
         default=False,
     )
 
+    transform_source: bpy.props.EnumProperty(
+        name='Transform Source',
+        description='Method of specifying $origin and $scale.\nEither manually specified in this panel, or via an object',
+        items=[
+            ('MANUAL', 'Manual Input', 'Specify the transforms manually in this panel'),
+            ('OBJECT', 'Object', 'Use an Object\'s transforms\nIf it is None, then no transforms are used'),
```

Comment:
Oh right yeah, make this Object have a small o as well (in the description, third string).
And I'd replace "if it is None" with "if it is not set", because None is programmer talk, and not everyone knows Python.

Reply:
Ah, fair point. I'll do this now alongside the - sign to make things consistent for testing.